ably-ios/ARTDataQuery+Private.h

```diff
@@ -16,4 +17,10 @@ ART_ASSUME_NONNULL_BEGIN
 
 @end
 
+@interface ARTRealtimeHistoryQuery ()
+
+@property (strong, readwrite) ARTRealtimeChannel *realtimeChannel;
```

Comment:
Hmm, why? realtimeChannel doesn't hold references to the ARTRealtimeHistoryQuery. There's no cycle here as far as I can see.